usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -207,6 +207,13 @@ def get_or_create_program_activity(row, submission_attributes):
     if prg_activity is None and row['program_activity_code'] is not None:
         prg_activity = RefProgramActivity.objects.create(**filters)
         logger.warning('Created missing program activity record for {}'.format(str(filters)))
+    # If the PA has a blank name, update it with the value in the row.
+    # PA loader should overwrite the names for the unique PAs from the official
+    # domain values list if the title needs updating, but for now grab it from the submission
+    if prg_activity and not prg_activity.program_activity_name:
```

Comment:
Since the list of PA data we originally load from the .csv (https://github.com/fedspendingtransparency/usaspending-api/blob/dev/usaspending_api/data/program_activity.csv) doesn't have any blank names, do we need this extra `if` statement? In other words, is it safe just to add the name when creating a new PA record up in line 208?

If you see existing records with an empty PA name in our data, those likely predate our updated PA load from the last sprint. We didn't clean those up as part of that work, choosing not to spend the time on a data migration that would fix up soon-to-be-deleted submissions. In retrospect, leaving those old, partial records there probably created some unnecessary confusion.